frontend/stylesheets/components/_reveal.scss

```diff
@@ -0,0 +1,146 @@
+$g: #60a415;
```

Comment:
Please don't use minimized variable names. All of this gets compiled out anyway, make them easy to know what they're for.